analyzer-models/src/test/java/ffb/test/utilities/GenericTests.java

```diff
@@ -0,0 +1,52 @@
+package ffb.test.utilities;
+
+import ffb.analyzer.models.espn.TransactionCounter;
+import org.junit.Assert;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class GenericTests {
+    public static void validateGetMethodsReturnNonNullValue(List<?> testObjects) throws InvocationTargetException, IllegalAccessException {
+        List<Method> getterMethods = new ArrayList<Method>();
+
+        for (Method method : testObjects.get(0).getClass().getMethods()) {
```

Comment:
How about `testObjects.get(0).getClass().getMethods().stream().filter(this::isGetterMethod).collect(Collectors.toList())`?

Reply:
Minor tweak to stream and used GenericTests instead of this since it's static, but roughly the same idea. Looks much cleaner.